cmd/bots/internal/telegram/handlers/handlers.go

```diff
@@ -154,6 +154,43 @@ func InitTgHandlers(environment *common.TelegramBotEnvironment, requestType chan
 			)
 		}
 		return RemoveNodeHandler(c, environment, requestType, responsePairType, args[0])
+
+	})
+
+	environment.Bot.Handle("/add_alias", func(c tele.Context) error {
+		args := c.Args()
+		if len(args) != 2 {
+			return c.Send(
+				messages.AliasWrongFormat,
+				&tele.SendOptions{
+					ParseMode: tele.ModeDefault,
+				},
+			)
+		}
+		return UpdateAliasHandler(c, environment, requestType, args[0], args[1])
+	})
+
+	environment.Bot.Handle("/aliases", func(c tele.Context) error {
+		nodes, err := messaging.RequestAllNodes(requestType, responsePairType)
+		if err != nil {
+			environment.Zap.Error("failed to get nodes list", zap.Error(err))
```

Comment:
I think in case of error user should be notified that's something wrong. What do you think about `return` in case of error? I think that is a good idea. (This can be applied to all telegram and discord handlers)

Reply:
Consider creating new PR for that.

Reply:
done